github/pulls.go

```diff
@@ -33,6 +33,7 @@ type PullRequest struct {
 	Merged              *bool      `json:"merged,omitempty"`
 	Mergeable           *bool      `json:"mergeable,omitempty"`
 	MergedBy            *User      `json:"merged_by,omitempty"`
+	MergeCommitSHA      *string    `json:"merged_commit_sha,omitempty"`
```

Comment:
I think you meant "merge_commit_sha", not "merge~~d~~_commit_sha":

![image](https://cloud.githubusercontent.com/assets/1924134/26229422/a67bf52c-3c0f-11e7-9c64-b99cb3ffc92e.png)